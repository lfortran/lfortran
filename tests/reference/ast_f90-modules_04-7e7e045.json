{
    "basename": "ast_f90-modules_04-7e7e045",
    "cmd": "lfortran --show-ast-f90 --no-color {infile}",
    "infile": "tests/../integration_tests/modules_04.f90",
    "infile_hash": "b04f80d4de5baea50808e962f08564088751c24a464b3d1d7c0c316f",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast_f90-modules_04-7e7e045.stdout",
    "stdout_hash": "1b6c92c555140732bc7b56428c1086bb0205deed13c32d4f3c7faf80",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}