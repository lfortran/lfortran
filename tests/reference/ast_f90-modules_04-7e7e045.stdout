module modules_04_a
implicit none


contains

subroutine b()
print *, "b()"
end subroutine b


end module modules_04_a


program modules_04
use, intrinsic :: iso_fortran_env
implicit none

call f()


contains

subroutine f()
use modules_04_a, only: b
call b()
end subroutine f


integer function g()
use modules_04_a, only: b
call b()
g = 5
end function g


end program modules_04
